channeldb/revocation_log.go

```diff
@@ -482,9 +482,9 @@ func writeTlvStream(w io.Writer, s *tlv.Stream) error {
 	return nil
 }
 
-// readTlvStream is a helper function that decodes the tlv stream from the
+// ReadTlvStream is a helper function that decodes the tlv stream from the
 // reader.
-func readTlvStream(r io.Reader, s *tlv.Stream) (tlv.TypeMap, error) {
+func ReadTlvStream(r io.Reader, s *tlv.Stream) (tlv.TypeMap, error) {
```

Comment:
When would we need these vs the normal/existing main entry point of stream creation?

Reply:
Removed this commit as it's no longer needed.